app/app.go

```diff
@@ -317,6 +317,14 @@ func New(
 		return initialVM, nil
 	})
 
+	app.UpgradeKeeper.SetUpgradeHandler("v0.5", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("Handler for upgrade plan: v0.5")
+
+		app.Migration05(ctx)
```

Comment:
I suggest using the approach from this guide:
https://docs.cosmos.network/master/building-modules/upgrade.html

Reply:
Yes, agreed and like I mentioned before!